Cargo.toml

```diff
@@ -46,6 +46,7 @@ serde_json = "1"
 sha2 = "0.10.1"
 thiserror = "1.0.30"
 url = "2.2.2"
+crc32fast = "1"
```

Comment:
Do we want to add the different hashes as optional dependencies? FWIW, the current design will degrade to no checksuming (instead of failing the parsing) if the configured hash function is not available

Reply:
I think so. For example, in Deno we have no need for this dep atm

Reply:
I've made them all optional, and kept the "sha256" feature as default for backwards compatibility.